src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -231,7 +232,7 @@ export const getModuleDetailsUma = async (
         maxRange,
         ({ start, end }: { start: number; end: number }) => {
           return oracleContract.queryFilter(
-            oracleContract.filters.AssertionMade(assertionId),
+            oracleContract.filters.AssertionMade(),
```

Comment:
had to remove assertion id here, because once a proposal is executed and settled, the assertion id becomes 0x0 which makes it impossible to find if filtering by this